workflow.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Chat GPT PR reviewer
```

Comment:
there is already a github workflow running, why adding a new one here?

Reply:
I was using it for testing purpose. I'll delete this file.